tools/editor/project_manager.cpp

```diff
@@ -75,15 +75,22 @@ class NewProjectDialog : public ConfirmationDialog {
 	String zip_title;
 	AcceptDialog *dialog_error;
 
-	bool _test_path() {
+	String _test_path() {
 
 		error->set_text("");
 		get_ok()->set_disabled(true);
 		DirAccess *d = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
-		if (project_path->get_text() != "" && d->change_dir(project_path->get_text())!=OK) {
+		String valid_path;
+		if (d->change_dir(project_path->get_text())==OK){
```

Comment:
Will `d->change_dir("")` behave as expected since we're no longer checking that `project_path->get_text()` is not empty?